bleak/backends/corebluetooth/client.py

```diff
@@ -190,20 +186,22 @@ async def get_services(self, **kwargs) -> BleakGATTServiceCollection:
            A :py:class:`bleak.backends.service.BleakGATTServiceCollection` with this device's services tree.
 
         """
-        if self._services is not None:
-            return self.services
+        if self._services:
+            return self._services
+
+        services = BleakGATTServiceCollection()
 
         logger.debug("Retrieving services...")
-        services = await self._delegate.discover_services()
+        cbt_services = await self._delegate.discover_services()
```

Comment:
CB is the common prefix for CoreBluetooth, so `cb_services` seems like the obvious name here.